### ISSUE TITLE
Definition MAX_CA_OBJECTS and WATER_OBJECT

### ISSUE THREAD

**Issue by AbyssMorgan** [Pottus/ColAndreas#21]
Hello can change the definitions of limits to the colandreas.inc in such a way:

// ColAndreas currently supports 50000 user collision objects (This can be raised as needed)
#if !defined MAX_CA_OBJECTS
	#define         MAX_CA_OBJECTS          10000
#endif

// Water object ID
#if !defined WATER_OBJECT
	#define         WATER_OBJECT            20000
#endif

And lack of a definition COLANDREAS_VERSION

Example for 1.4.0 version:
#define COLANDREAS_VERSION (10400) //a.b.c 10000\*a+100\*b+c

**Comment by Crayder** [comment]
MAX_CA_OBJECTS can't be changed by the PAWN script, you'd have to edit the plugin to change it. 

Same goes for the water object, but why would anybody ever need to change it anyways? 20000 can't be used by SA-MP for objects.

The CA version definition is reasonable.

**Comment by AbyssMorgan** [comment]
As for the reduction of these values, and do not increase, at our discretion.

Example:
// ColAndreas currently supports 50000 user collision objects (This can be raised as needed)
#define MAX_CA_OBJECTS 1000
#include \<ColAndreas\>

Mistake WATER_OBJECT is not a max value not to change :P

**Comment by Crayder** [comment]
I'm pretty sure you could do this...

```
#if defined MAX
    #if MAX > 50000
        #error max is too high
    #endif
#else
   #define MAX 10000

**Comment by Crayder** [comment]
I'm on my phone, not home, so if you're willing to feel free to open a pull request.

**Comment by Crayder** [comment]
Btw, on GitHub you can say things like "closes #21" or "solves #21" on pull requests and the issue will automatically close when it's pulled.
